templates/security/disa-stig.html

```diff
@@ -86,7 +86,7 @@ <h2>
           <a href="/security/contact-us"
              class="p-button--positive js-invoke-modal">Contact us</a>
           <a href="https://www.brighttalk.com/webcast/6793/604776"
-             aria-label="Watch the 'How does Ubuntu enable your compliance with FIPS, and DISA-STIG?' webinar">Watch the webinar&nbsp;&rsaquo;</a>
+             aria-label="Watch the 'How does Ubuntu enable your compliance with FIPS, and DISA-STIG?' webinar">Watch the webinar &nbsp;&rsaquo;</a>
```

Comment:
```suggestion
             aria-label="Watch the 'How does Ubuntu enable your compliance with FIPS, and DISA-STIG?' webinar">Watch the webinar&nbsp;&rsaquo;</a>
```
No need for a space here as `&nbsp;` is a space